components/NavigationBar/tests/unit/NavigationBarButtonTitleCasingTests.swift

```diff
@@ -0,0 +1,135 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import XCTest
+import MaterialComponents.MaterialNavigationBar
+import MaterialComponents.MaterialButtons
+
+class NavigationBarButtonTitleCasingTests: XCTestCase {
+
+  var navigationBar: MDCNavigationBar!
```

Comment:
Also why not `let navigationBar = MDCNavigationBar()` and get rid of `setUp()`.

Reply:
We need this to be instantiated for every test. Doing a let would result in us reusing the same instance for every test.